staging/src/k8s.io/client-go/tools/clientcmd/config.go

```diff
@@ -383,6 +390,11 @@ func writeCurrentContext(configAccess ConfigAccess, newCurrentContext string) er
 			return err
 		}
 		currConfig.CurrentContext = newCurrentContext
+
+		if lockErrors[file] != nil {
```

Comment:
move this above the getConfigFromFile call

Reply:
Fixed in ecf3b83e88ef4c4791889f922514cf459d3b3058